pkg/controllers/hypernode/configmap_handler.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2025 The Volcano Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hypernode
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	coreinformers "k8s.io/client-go/informers/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog/v2"
+
+	"volcano.sh/volcano/pkg/controllers/hypernode/config"
+)
+
+func (hn *hyperNodeController) setConfigMapNamespaceAndName() {
+	namespace := os.Getenv(config.NamespaceEnvKey)
+	if namespace == "" {
+		namespace = config.DefaultNamespace
+	}
+	releaseName := os.Getenv(config.ReleaseNameEnvKey)
+	if releaseName == "" {
+		releaseName = config.DefaultReleaseName
+	}
+	hn.configMapNamespace = namespace
+	hn.configMapName = releaseName + "-controller-configmap"
```

Comment:
A new `networkTopologyDiscovery` configuration is added, but we add a new configmap at the controller letvel, is this appropriate? If the controller needs to add configurations of other fields in the future, should we also use this configmap? Currently, if any data of the configmap changed, it will trigger the syncHandler. So I think it is necessary to refine the logic of enqueueConfigmap that syncHandler should be triggered only when `networkTopologyDiscovery` field changes.